.travis.yml

```diff
@@ -29,9 +29,9 @@ install:
   # -----------------
   - export CONDA_BASE=http://repo.continuum.io/miniconda/Miniconda
```

Comment:
Yes. In fact I am surprised that the `wget` worked here. I spent my whole weekend fixing that in mane places in conda-forge :smile: